src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorDescriptorTest.groovy

```diff
@@ -146,27 +148,37 @@ abstract class IqPolicyEvaluatorDescriptorTest
   def 'it validates that application items are filled'() {
     setup:
       def descriptor = getSelectedApplication()
-      GroovyMock(IqUtil, global: true)
       def job = Mock(Job)
+      ListBoxModel appList
 
     when:
-      descriptor.doFillApplicationIdItems('', job)
+      appList = descriptor.doFillApplicationIdItems('', job)
 
     then:
-      1 * IqUtil.doFillIqApplicationItems('', job)
+      appList.size() == 1
+      appList.get(0).name == FormUtil.EMPTY_LIST_BOX_NAME
+      appList.get(0).value == FormUtil.EMPTY_LIST_BOX_VALUE
   }
 
   def 'it uses custom credentials for application items'() {
     setup:
       def descriptor = getSelectedApplication()
-      GroovyMock(IqUtil, global: true)
+      // create a config so that IqUtil.doFillApplicationIdItems will go down the path that uses custom credentials
+      GlobalNexusConfiguration.globalNexusConfiguration.iqConfigs = [new NxiqConfiguration('http://server/url',
+          null)]
       def job = Mock(Job)
 
     when:
       descriptor.doFillApplicationIdItems('credentialsId', job)
 
     then:
-      1 * IqUtil.doFillIqApplicationItems('credentialsId', job)
+      // this is a bit odd, but because we're mixing Groovy and Java classes SelectedApplication, we can't
```

Comment:
Probably nice to have a factory for the IqUtil to facilitate this kind of thing (nothing needed now though).